utils/tests/test_mixins.py

```diff
@@ -190,20 +194,26 @@ def test_project_work_percentage_mixin_should_call_get_projects_work_percentage_
     ):
         current_date = timezone.now().date()
 
-        with patch("users.models.CustomUser.get_projects_work_percentage") as get_projects_work_percentage:
+        with patch(
+            "users.models.CustomUser.get_projects_work_hours_and_percentage"
+        ) as get_projects_work_hours_and_percentage:
             response = self.view(self.request, year=current_date.year, month=current_date.month)
 
         self.assertEqual(response.status_code, 200)
-        get_projects_work_percentage.assert_called_once_with(current_date - timezone.timedelta(days=30), current_date)
+        get_projects_work_hours_and_percentage.assert_called_once_with(
+            timezone.now().date().replace(year=current_date.year, month=current_date.month, day=1), current_date
```

Comment:
Instead of `timezone.now().date.replace(...)` use `timezone.datetime(...).date()`

Reply:
Done.